mitodl/open-discussions
# Refactored channels/views*.py into separate modules

#### What are the relevant tickets?
N/A

#### What's this PR do?
Refactors `channels/views*.py` into separate modules because the files were getting lengthy.

#### How should this be manually tested?
App should function as expected. Test coverage should not change. In the diff you should observe `cassettes/*` and `factor_data/*`files being renamed only (no new files or modifications).
